src/app/static/src/app/store/baseline/mutations.ts

```diff
@@ -26,7 +27,7 @@ export const mutations: MutationTree<BaselineState> & BaselineMutations = {
         const data = action.payload;
         if (data) {
             state.country = data.data.country;
-            state.pjnz = data;
+            state.pjnz = Object.freeze(data);
```

Comment:
A more radical re-architecture would be to keep this big objects in our own local cache, outside of vue, and just keep an index over them in the store. I feel like that might be slightly nicer, but it would be a more sizable refactor, so I think just something to consider at a later date.

Reply:
Agreed. It's probably worth using this frozen object technique for a bit to see what the use cases are, to find a reusable pattern for using frozen state with dynamic state

Reply:
Yeah, also not totally sure how this will translate when making components reusable as this relies on the object being passed to the component being frozen already.